README.md

```diff
@@ -96,6 +119,28 @@ Reviews PHP follows [semantic versioning](https://semver.org/) specifications.
 The MIT License (MIT). Please see [License File](https://github.com/Pod-Point/reviews-php/LICENCE) for more information.
 
 ## Development
+
+### Caching
+
+Using this library with Laravel and to have it compatible, a **LaravelCacheAdapter** has been added. To cache Requests response contents, extend the Request class using the **AbstractCacheableRequest** class, this will automatically cache the responses.
```

Comment:
not sure what you mean with this first sentence

Reply:
I have removed the first sentence, not required here since I have mentioned in the Laravel Caching. What I wanted to say is that to keep this library compatible with laravel a `LaravelCacheAdapter` has been added.